suse/rdma-core.spec

```diff
@@ -325,6 +330,14 @@ rdma-ndd is a system daemon which watches for rdma device changes and/or
 hostname changes and updates the Node Description of the rdma devices based
 on those changes.
 
+%package -n python3-pyverbs
+Summary: Python3 API over IB verbs
```

Comment:
Minor incoherency in indent here. Nothing blocking though

Reply:
fixed